program/idl.json

```diff
@@ -9265,4 +9338,4 @@
       ]
     }
   ]
-}
+}
```

Comment:
Please revert that change. 🙏

Reply:
I think this might be a bug in github. I reverted the idl.json, but the change was still there. It looks like the ending bracket was moved to a new line due to the new instruction.

Reply:
Okay no worries, this should auto-fix itself in another PR.